src/pages/en/guides/styling.md

```diff
@@ -84,7 +84,7 @@ const backgroundColor = "rgb(24 121 78)";
 <h1>Hello</h1>
 ```
 
-Any *serializable* front matter variable is supported, including props passed to your component through `Astro.props`.
+📚 See our [directives reference](/en/reference/directives-reference#definevarsvariables) page to learn more about `define:vars`.
```

Comment:
Just pointing out that earlier on this page, in Global styles, we refer to global as an attribute, not a directive, and we don't link to the directives page. (Side note: maybe the Global header should be one level higher, so that it shows up in the sidebar with scoped?) Should this be treated the same way?

Reply:
I think that it's better to use the correct terminology here, and on one of the first pages explain what a directive is (but that's out of the scope of this PR)